examples/lsc/mag240m/gnn.py

```diff
@@ -234,9 +237,15 @@ def configure_optimizers(self):
                     num_layers=len(args.sizes), dropout=args.dropout)
         print(f'#Params {sum([p.numel() for p in model.parameters()])}')
         checkpoint_callback = ModelCheckpoint(monitor='val_acc', mode = 'max', save_top_k=1)
-        trainer = Trainer(gpus=args.device, max_epochs=args.epochs,
-                          callbacks=[checkpoint_callback],
-                          default_root_dir=f'logs/{args.model}')
+        try:
+          trainer = Trainer(gpus=args.device, max_epochs=args.epochs,
+                            callbacks=[checkpoint_callback],
+                            default_root_dir=f'logs/{args.model}')
+        except:
+          # up to date usage
+          trainer = Trainer(devices=len(args.device.split(',')), max_epochs=args.epochs,
```

Comment:
why only taking len()? should device specify gpu ids?

Reply:
devices seems to just be the number of gpus u want to use, not the specific ids
<img width="853" alt="Screen Shot 2023-04-04 at 8 44 46 AM" src="https://user-images.githubusercontent.com/20074092/229846338-f556da2d-bbeb-45c7-bce2-ab3c6cbd679d.png">